src/plone/restapi/services/site/get.py

```diff
@@ -54,6 +59,11 @@ def __call__(self, expand=False):
             }
         )
 
+        if HAS_MULTILINGUAL and IPloneAppMultilingualInstalled.providedBy(self.request):
+            result["site"]["plone.is_multilingual"] = True
+        else:
+            result["site"]["plone.is_multilingual"] = False
+
```

Comment:
Which was your intention for the `features` dict? Should we put this one on it?

Reply:
@sneridagh I would prefer to add it to the `features` dict.

I don't like that we used the `plone` prefix here; let's not make the problem worse by using it for things that don't even come from a registry setting with the same name.

And, this is exactly what the features dict is for: when the frontend needs to be able to check if some feature is provided by the backend.